PowerPointLabs/PowerPointLabs/Settings.StyleCop

```diff
@@ -274,22 +274,22 @@
         </Rule>
         <Rule Name="SystemUsingDirectivesMustBePlacedBeforeOtherUsingDirectives">
```

Comment:
I understand, however wouldn't it be better to be explicit?

@damithc What's your opinion on removing vs having the entry (because by default, if the entry doesn't exist on the Settings file, it's enabled)